source/server/config/network/mongo_proxy.h

```diff
@@ -14,10 +16,16 @@ namespace Configuration {
  * Config registration for the mongo proxy filter. @see NamedNetworkFilterConfigFactory.
  */
 class MongoProxyFilterConfigFactory : public NamedNetworkFilterConfigFactory {
+  NetworkFilterFactoryCb
```

Comment:
if this is meant to be private, should go in private section below public section.